src/main/java/ch/uzh/ifi/hase/soprafs24/controller/TeamController.java

```diff
@@ -101,4 +101,20 @@ public TaskGetDTO createTask(@PathVariable("ID") Long teamId,
     // convert internal representation of task back to API
     return DTOMapper.INSTANCE.convertEntityToTaskGetDTO(createdTask);
   }
+
+  @GetMapping("/teams/{ID}/tasks")
+  @ResponseBody
+  public List<TaskGetDTO> getTasks(
+      @PathVariable("ID") Long teamId, @RequestHeader("Authorization") String token) {
+    // check if user is authorized (valid token)
+    authorizationService.isAuthorized(token);
```

Comment:
Optional Improvement: use new `isAuthorizedAndInTeam` auth function

Reply:
all right, I will look in next merge, thanks